src/test/java/nextstep/ladder/domain/LadderLineTest.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.ladder.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class LadderLineTest {
+    @Test
+    public void init() {
+        LadderLine ladderLine = LadderLine.of(2, ()-> true);
```

Comment:
코딩 컨벤션에 따라 람다 사용 시 공백을 추가해보면 좋을 것 같습니다.
아래 24라인도 같이 확인해주시면 좋을 것 같아요.
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
```suggestion
        LadderLine ladderLine = LadderLine.of(2, () -> true);
```